kaska/s1_observations.py

```diff
@@ -32,9 +32,9 @@
 
 def get_s1_dates(s1_file):
     """Gets the dates from a LMU processed netCDF Sentinel 1 file"""
-    times = [float(s1_file.GetRasterBand(b+1).GetMetadata()['NETCDF_DIM_time']) 
-                                for b in range(s1_file.RasterCount)]
-    times = [dt.datetime(1970,1,1) + dt.timedelta(days=x) for x in times ]
+    times = [float(s1_file.GetRasterBand(b+1).GetMetadata()['NETCDF_DIM_time'])
+             or b in range(s1_file.RasterCount)]
```

Comment:
This still needs to be changed to a `for`, doesn't it?

Reply:
Oops sorry yes I will correct and then resubmit the PR